src/frontend/src/lib/components/dapps/DappsCarouselSlide.svelte

```diff
@@ -0,0 +1,38 @@
+<script lang="ts">
+	import Img from '$lib/components/ui/Img.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+	import { modalStore } from '$lib/stores/modal.store';
+	import { type CarouselSlideOisyDappDescription } from '$lib/types/oisyDappDescription';
+	import { replacePlaceholders } from '$lib/utils/i18n.utils';
+
+	export let dappsCarouselSlide: CarouselSlideOisyDappDescription;
+	$: ({
+		carousel: { text, callToAction },
+		logo,
+		name
+	} = dappsCarouselSlide);
+</script>
+
+<div class="flex">
+	<div class="mr-4 shrink-0">
+		<Img
```

Comment:
can you use `Logo` component here? Or it does not fit the needs?

Reply:
`Logo` has some additional stuff that does not fit to this case